tests/phpunit/CRM/Core/I18n/SchemaTest.php

```diff
@@ -40,9 +40,17 @@ public function tearDown() {
    * @param string $expectedRewrite
    *
    * @dataProvider translateTables
+   * @throws \CRM_Core_Exception
    */
   public function testI18nSchemaRewrite($table, $expectedRewrite) {
     CRM_Core_I18n_Schema::makeMultilingual('en_US');
+    $domains = $this->callAPISuccess('Domain', 'get')['values'];
+    foreach ($domains as $domain) {
```

Comment:
Let's make sure we are actually testing multiple domains:

```suggestion
    $domains = $this->callAPISuccess('Domain', 'get')['values'];
    $this->assertGreaterThan(1, count($domains));
    foreach ($domains as $domain) {
```